public/app/features/backup/components/DBIcon/DBIcon.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react';
+import { toPascalCase } from '@grafana/data';
+import { DBIconProps } from './DBIcon.types';
+import * as Icons from './assets';
+
+export const DBIcon: FC<DBIconProps> = ({ type, size, ...rest }) => {
+  //@ts-ignore
```

Comment:
Is this really required? Can't we just cast the return of `Icons` or the return of `toPascalCase` depending on the error we get.

Reply:
seems like TS really doesn't like wildcard imports (unless casting `Icon` as `any`), so I went the other way around with a map, to play safe.